src/__internal__/focus-trap/focus-trap.component.js

```diff
@@ -157,10 +166,15 @@ const FocusTrap = ({
     ) {
       // the trap breaks if it tries to refocus a disabled element
       setElementFocus(currentFocusedElement);
+    } else if (
+      wrapperRef?.current &&
+      wrapperRef.current.hasAttribute("tabindex")
```

Comment:
Agreed. Actually I thought that line already existed in the code before I changed anything here but it seems that's not the case - regardless I agree it might as well be written the more modern and readable way.